crates/polytune-server-core/src/state.rs

```diff
@@ -175,11 +187,16 @@ enum PolicyStateKind<C> {
         typed_program: TypedProgram,
         client: C,
     },
+    // All consts have been received, but the actual mpc computation has not started yet
     Running {
         policy: Policy,
         channel: Channel<C>,
     },
+    // mpc computation is executing in a separate tokio task
     Executing {
+        // use Notify because we notify in both directions, first from the `cancel` method
+        // to the tokio task to signale cancellation, and then the other direction if the
```

Comment:
```suggestion
        // to the tokio task to signal cancellation, and then the other direction if the
```

I think it is just signal.